build/repo.props

```
@@ -125,6 +127,10 @@
                           @(ProjectToBuild);
                           @(ProjectToExclude);" />
 
+        <!--
+          Use caution to avoid deep recursion. If the globbing pattern picks up something which exceeds MAX_PATH,
+          the entire pattern will silently fail to evaluate correctly.
```

Is there any way we can make it fail loudly?